docs/source/tutorials/outliers.ipynb

```diff
@@ -0,0 +1,629 @@
+    "\n",
+    "**Overview of what we'll do in this tutorial:**\n",
+    "\n",
+    "- Load the [cifar10](https://www.cs.toronto.edu/~kriz/cifar.html) dataset and do some basic data pre-processing.\n",
+    "- Create `train_data` and `test_data` such that `train_data` only contains animals and `test_data` contains all classes.\n",
+    "- Load pretrained `timm` model and extract `train_data` and `test_data` feature embeddings.\n",
+    "- Use `cleanlab` to find naturally occuring outlier examples in the `train_data`.\n",
+    "- Use `cleanlab` to find outlier examples (non-animals) in the `test_data`.\n",
+    "- Explore threshold selection for labeling outliers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "869e24c3",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Quickstart\n",
+    "<br/>\n",
+    "    \n",
+    "Do you already have `feature_embeddings` from your data? Run the code below to generate outlier scores.\n",
+    "\n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
+    "    \n",
+    "```python\n",
+    "\n",
+    "from sklearn.neighbors import NearestNeighbors\n",
+    "from cleanlab.rank import get_outlier_scores\n",
+    "\n",
+    "# To get outlier scores for train_data using train_feature_embeddings\n",
+    "outlier_scores = get_outlier_scores(features=train_feature_embeddings)\n",
+    "\n",
+    "# To get outlier scores for test_data using feature matrix test_feature_embeddings\n",
+    "knn = NearestNeighbors(n_neighbors=10, metric=\"cosine\").fit(train_feature_embeddings)\n",
+    "outlier_scores = get_outlier_scores(features=test_feature_embeddings, knn=knn)\n",
+    "    \n",
+    "\n",
+    "```\n",
+    "    \n",
+    "</div>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e29a3608",
+   "metadata": {},
+   "source": [
+    "## 1. Install the required dependencies\n",
+    "You can use `pip` to install all packages required for this tutorial as follows:\n",
+    "\n",
+    "```ipython3\n",
+    "!pip install cleanlab\n",
+    "...\n",
+    "# Make sure to install the version corresponding to this tutorial\n",
+    "# E.g. if viewing master branch documentation:\n",
+    "#     !pip install git+https://github.com/cleanlab/cleanlab.git\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "48cb3aed",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# Package installation (hidden on docs website).\n",
+    "# If running on Colab, may want to use GPU (select: Runtime > Change runtime type > Hardware accelerator > GPU)\n",
+    "# Package versions we used: matplotlib==3.5.1, numpy==1.21.6, torch==1.11.0, scikit-learn==1.0.2, torchvision==0.12.0, timm==0.5.4, cleanlab==2.0.0\n",
+    "\n",
+    "dependencies = [\"matplotlib\", \"torch\", \"torchvision\", \"sklearn\", \"timm\"]\n",
```

Comment:
BTW we generally want to include `cleanlab` in the dependencies variable.
The cell below will check if each of the dependencies has been installed and tell the user they should install it if not (so would be good to also check this for cleanlab)